.github/workflows/ci.yml

```diff
@@ -95,7 +95,7 @@ jobs:
         with:
           environment-file: environment.yml
           cache-downloads: true
-
+          
```

Comment:
Remove the added blank spaces, there is no actual change wrt the file on master.

Reply:
Done